scripts/us_census/acs5yr/subject_tables/S2703/S2703_spec.json

```diff
@@ -0,0 +1,219 @@
+{
+    "preprocess": {
+        "find_and_replace": {
+            "Estimate!!Total!!Civilian noninstitutionalized population!!COVERAGE ALONE OR IN COMBINATION!!Employer-based health insurance alone or in combination": "Estimate!!Total!!Civilian noninstitutionalized population",
```

Comment:
Is the token `Employer-based health insurance alone or in combination` being dropped?

This has a PV mapping in the confing below:
```
         "Employer-based health insurance alone or in combination": "WithEmployerBasedHealthInsurance"
```

Reply:
The token is only removed for the `Total` columns where the health insurance PV does not apply.  The token `Employer-based health insurance alone or in combination` applies for the `Private Coverage` set of columns.

![Screen Shot 2022-05-21 at 10 11 33 AM](https://user-images.githubusercontent.com/33577829/169635739-1765f249-7c03-4cd7-8fde-fbe8815f8792.png)